packages/edit-site/src/components/list/added-by.js

```diff
@@ -77,18 +81,26 @@ function AddedByAuthor( { id } ) {
 	] );
 	const [ isImageLoaded, setIsImageLoaded ] = useState( false );
 
+	const hasAvatar = !! user?.avatar_urls?.[ 48 ];
```

Comment:
I think we could just create a variable here and reference that variable in JSX.

```suggestion
	const avatarURL = user?.avatar_urls?.[ 48 ];
	const hasAvatar = !! avatarURL;
```

So that we won't repeat ourselves :).